lib/v2/gcores/radar.js

```diff
@@ -15,5 +15,21 @@ module.exports = {
                 target: '/gcores/tag/:tag',
             },
         ],
+        radios: [
+            {
+                title: '播客',
+                docs: 'https://docs.rsshub.app/new-media.html#ji-he-wang-bo-ke',
+                source: ['/'],
+                target: '/gcores/radios',
+            },
+        ],
+        categories: [
+            {
+                title: '播客-分类',
+                docs: 'https://docs.rsshub.app/new-media.html#ji-he-wang-bo-ke',
+                source: ['/:category'],
+                target: '/gcores/radios/:category',
+            },
+        ],
```

Comment:
These two rules will match `radios.gcores.com` and `categories.gcores.com` respectively and I don't think these two domain exists